src/content/self-hosted/gke.mdx

```diff
@@ -14,24 +14,23 @@ In order to fully install Okteto, you will need the following:
 - A Kubernetes cluster
 - A working installation of kubectl
 - A working installation of Helm v3
-- A Google (or GitHub) OAuth application
-- A Google Cloud service account and a key
-- A Google Cloud Storage bucket
 - An Okteto License (optional)
 
 ### Subdomain
 
-You'll need access to a subdomain to which you can add a wildcard DNS record, such as dev.example.com.
+You'll need access to a [subdomain](self-hosted/administration/configuration.mdx#subdomain) to which you can add a wildcard DNS record, such as dev.example.com.
+By default, all endpoints created by Okteto for your development environments will be exposed in this wildcard subdomain.
 
-This guide will assume that your domain is registered in Google's [Cloud DNS](https://cloud.google.com/dns). Other DNS services can be used as well, but are not covered by this guide.
+This guide will assume that your domain is registered in Google's [Cloud DNS](https://cloud.google.com/dns).
+Other DNS services can be used as well, but are not covered by this guide.
```

Comment:
```suggestion
You can use any DNS service you prefer, but this guide focuses specifically on Google's Cloud DNS.
```

We might even consider using the admonitions `info` callout here just to make sure this is obvious :)